app/controllers/spina/admin/registers_controller.rb

```diff
@@ -60,8 +60,9 @@ def set_breadcrumb
       end
 
       def set_government_organisations
-        register_data = RegistersClientWrapper.registers_client.get_register('government-organisation', 'beta')
-        @government_organisations = register_data.get_records
+        government_organisation_register = Register.find_by(slug: 'government-organisation')
+        government_organisation_records = Record.where(register_id: government_organisation_register.id, entry_type: 'user')
```

Comment:
Can we just call the records from the register? i.e. 
`government_organisation_records  = Register.find_by(slug: 'government-organisation').records.where(entry_type: "user")`